providers/ionoscloud/certificate.go

```diff
@@ -14,15 +14,15 @@ type CertificateGenerator struct {
 
 func (g *CertificateGenerator) InitResources() error {
 	client := g.generateClient()
-	cloudAPIClient := client.CertificateManagerAPIClient
+	certManagerAPIClient := client.CertificateManagerAPIClient
 	resourceType := "ionoscloud_certificate"
 
-	response, _, err := cloudAPIClient.CertificatesApi.CertificatesGet(context.TODO()).Execute()
+	response, _, err := certManagerAPIClient.CertificatesApi.CertificatesGet(context.TODO()).Execute()
 	if err != nil {
 		return err
 	}
 	if response.Items == nil {
-		log.Printf("[WARNING] 'nil' expected a response containing certificates but received 'nil' instead.")
+		log.Printf("[WARNING] expected a response containing certificates but received 'nil' instead.\n")
```

Comment:
you can probably use log.Println?

Reply:
@cristiGuranIonos I modified all messages and I removed the newline based on what we discussed: `log.Printf` doesn't require a newline at the end, it adds it anyways.